ui/src/core/forms.js

```diff
@@ -1878,9 +1878,6 @@ window.forms = {
 
       // Add the options
       $.each(effects, function(_index, element) {
-        if (effectsType !== 'all' && element.group !== effectsType) {
```

Comment:
I thought this condition was here so that the `variant` of any effects property filtered the effects returned.

Reply:
Yes, this condition should filter the effects based on the `variant` but having this line prevents us from showing marquee effects as previously we were setting the effects field manually for elements.

`properties-panel.js`
![image](https://github.com/xibosignage/xibo-cms/assets/128448242/20a4ef5e-a61e-473c-8e6b-8aed291eb155)

Reply:
Should we change it's `variant = all`?

Reply:
Sorry I don't understand - does variant filtering still work without this condition (via some other mechanism)?

Reply:
No, it won't work anymore without this condition. However, this condition prevents showing marquee effects.

Reply:
Breaking something else can't be the solution, so perhaps the variant logic needs some refactoring. We have to be able to control which effects are returned for the property - either all, paged only or non-paged only (marquee). Please can you discuss with Mauro?

Reply:
Yes, will do.